src/main/java/frc/auto/AutoBuilder.java

```diff
@@ -55,7 +55,8 @@ public void recordData() {
 
     public void convertData() {
         try {
-            writer.setFileName("test");
+            System.out.println("Made it to convertData()!");
+            writer.setFileName("test" + Constants.version);
```

Comment:
Why only test? this should be set by the dashboard or a constant

Reply:
This will be changed once we decide how the autonomous selector will work!